introduction/es/06-conclusion.asciidoc

```diff
@@ -0,0 +1,14 @@
+== Conclusiones
+
+En este módulo hemos visto una introducción a InnerSource.
+InnerSource aplica las buenas prácticas de desarrollo de los proyectos de software libre en desarrollos internos.
+Ofrece una opción adicional a los consumidores internos cuando los equipos que producen software no son capaces de llevar a cabo una nueva funcionalidad requerida.
+Para llevar a cabo InnerSource de forma satisfactoria, se ha de involucrar al https://innersourcecommons.org/learn/learning-path/product-owner/01[_product owner_] y a los https://innersourcecommons.org/learn/learning-path/trusted-committer/01[_trusted committers_] del *equipo anfitrión* así como a https://innersourcecommons.org/learn/learning-path/contributor/01[_contribuidores_] del *equipo invitado*
+Hecho de forma efectiva hará que InnerSource traiga beneficios a ambas partes.
+Los principios clave sobre los que se basa el trabajo de InnerSource son las *contribuciones voluntarias de código fuente* y la *priorización de actividades de mentoría*.
+
+Mientras que este módulo contiene una visión general de InnerSource, hay muchos otros detalles que te resultarán útiles en hacer que InnerSource funcione realmente en tu equipo.
```

Comment:
Using a leading adjective - _"realmente funcione"_ instead of _"funcione realmente"_ -  implies that it is expected to work, while in contrast the trailing one seems to suggest that the mentioned details are not just a mere reinforcement, but actually needed, almost unavoidable. (Exaggerating a bit here for the sake of explanation ;-) ).

Reply:
not sure if you're commenting, or actually making a suggestion :)